pages/api/notes/search.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { prisma } from "@/lib/prisma";
+import { Note } from "@prisma/client";
+import { NoteSearchPreview } from "@/interfaces/noteSearchPreview";
+
+
+const TITLE_LIMIT: number = 50
+const CONTENT_LIMIT: number = 100
+
+function convertToNotePreview(notes: Note[], searchTerm: string): NoteSearchPreview[] {
+    return notes.map(note => {
+        return { id: note.id, title: note.title.substring(0, TITLE_LIMIT), content: note.content.substring(0, CONTENT_LIMIT), searchTerm: searchTerm }
+    })
+}
+
+export default async function handle(req: NextApiRequest, res: NextApiResponse) {
+    let searchTerm = "*"
+    if (req.query.searchTerm && typeof req.query.searchTerm === "string") {
```

Comment:
The `req.query.searchTerm` condition is redundant.

Reply:
done